packages/fluentui/react-builder/src/components/Canvas.tsx

```diff
@@ -336,8 +445,12 @@ export const Canvas: React.FunctionComponent<CanvasProps> = ({
               )}
               {inUseMode && <EventListener capture type="focus" listener={handleFocus} target={document} />}
               {renderJSONTreeToJSXElement(jsonTree, renderJSONTreeElement)}
+              {inUseMode && enabledVirtualCursor && (
+                <EventListener type="keydown" listener={handleKeyDown} target={document} />
+              )}
               <div style={{ bottom: '0', position: 'absolute' }}>
                 <ReaderNarration
```

Comment:
Virtual cursor navigates even to `ReaderNarration`. Is it is not part of the designed UI, it is not expected.

Reply:
Agreed via chat to leave it as it is.